docs/conf.py

```diff
@@ -241,13 +241,13 @@
     ]
 
 # If true, show URL addresses after external links.
-#man_show_urls = False
+# man_show_urls = False
 
 
 # Mock extra modules
 class Mock(MagicMock):
     @classmethod
-    def __getattr__(cls, name):
+    def __getattr__(cls, name: str) -> MagicMock:
```

Comment:
not 100% confident about this either

Reply:
> The correct would be `-> 'Mock'`.

You sure? To be honest, I find aliased imports like on top of this file very confusing:
`from unittest.mock import Mock as MagicMock`

Reply:
Yeah, function ends with `return Mock()`, and whatever the `Mock` might be, that's for mypy and linters to untangle.

Reply:
It's not great to be fair.  Also mypy doesn't run in the docs/ dir
```
# ruff .
docs/conf.py:250:40: F821 Undefined name `Mock`
Found 1 error.

# ruff rule F821
# undefined-name (F821)

Derived from the **Pyflakes** linter.
```
I suppose we can add noqa, but would not sleep well in the night :D